x/feemarket/spec/01_concepts.md

```diff
@@ -0,0 +1,20 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Base fee
+
+The base fee is a global base fee defined at the consensus level. It is adjusted for each block based on the total gas used in the previous block.
+
+Unlike the Cosmos SDK local `minimal-gas-prices`, this value is persisted in the KVStore which make a reliable value for validators to agree upon.
```

Comment:
Unlike the Cosmos SDK local `minimal-gas-prices`, this value is persisted in the KVStore which provides a reliable value for validators to agree upon.